CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+
+# Contributing Guide For tough-cookie
+
+This page lists the operational governance model of this project, as well as the recommendations and requirements for how to best contribute to tough-cookie. We strive to obey these as best as possible. As always, thanks for contributing – we hope these guidelines make it easier and shed some light on our approach and processes.
+
+## Governance Model
+
+The intent and goal of open sourcing this project is to increase the contributor and user base. The governance model is one where new project leads (`admins`) will be added to the project based on their contributions and efforts, a so-called "do-acracy" or "meritocracy" similar to that used by all Apache Software Foundation projects.
+
+## Issues, requests & ideas
+
+Use GitHub Issues page to submit issues, enhancement requests and discuss ideas. Please ensure your description is clear and has sufficient instructions to be able to reproduce the issue.
+
+### Bug Reports and Fixes
+
+- If you find a bug, please search for it in the [Issues](https://github.com/salesforce/tough-cookie/issues), and if it isn't already tracked,
+  [create a new issue](https://github.com/salesforce/tough-cookie/issues/new). Fill out the "Bug Report" section of the issue template. Even if an Issue is closed, feel free to comment and add details, it will still
+  be reviewed.
+- Issues that have already been identified as a bug (note: able to reproduce) will be labelled `bug`.
+- If you'd like to submit a fix for a bug, [send a Pull Request](#creating_a_pull_request) and mention the Issue number.
+- Include tests that isolate the bug and verifies that it was fixed.
+
+### New Features
+
+- If you'd like to add new functionality to this project, describe the problem you want to solve in a [new Issue](https://github.com/salesforce/tough-cookie/issues/new).
+- Issues that have been identified as a feature request will be labelled `enhancement`.
+- If you'd like to implement the new feature, please wait for feedback from the project
+  maintainers before spending too much time writing the code. In some cases, `enhancement`s may
+  not align well with the project objectives at the time.
+
+### Tests, Documentation, Miscellaneous
+
+- If you'd like to improve the tests, you want to make the documentation clearer, you have an
+  alternative implementation of something that may have advantages over the way its currently
+  done, or you have any other change, we would be happy to hear about it!
+- If its a trivial change, go ahead and [send a Pull Request](#creating_a_pull_request) with the changes you have in mind.
+- If not, [open an Issue](https://github.com/salesforce/tough-cookie/issues/new) to discuss the idea first.
+
+If you're new to our project and looking for some way to make your first contribution, look for
+Issues labelled `good first contribution`.
```

Comment:
I've added this to our [labels](https://github.com/salesforce/tough-cookie/labels).